docs/LSPSupport.md

```diff
@@ -370,6 +370,15 @@ Here is an example with the [Go Language Server](./user-defined-ls/gopls.md) sho
 
 ![textDocument/foldingRange](./images/lsp-support/textDocument_foldingRange.gif)
 
+### Selection range
+
+[textDocument/selectionRange](https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#textDocument_selectionRange) is implemented with
+the `extendWordSelectionHandler` extension point and used via the IDE's **Extend Selection** and **Shrink Selection** actions.

```

Comment:
Please add shortcut sample in () to execute those actions.

Reply:
Sorry, but what does "in ()" mean? Is there already a list of supported keyboard shortcuts for LSP4IJ-integrated actions?

Reply:
I wanted to say add (Ctrl+W) / (Ctrl+Shift+W)